content/cloud-docs/run/states.mdx

```diff
@@ -16,11 +16,13 @@ In the list of workspaces on Terraform Cloud's main page, each workspace shows t
 _States in this stage:_
 
 - **Pending:** Terraform Cloud hasn't started action on a run yet. Terraform Cloud processes each workspace's runs in the order they were queued, and a run remains pending until every run before it has completed.
+- **Fetching:** If Terraform Cloud has not yet fetched the configuration from VCS, the run will go into this state until the configuration is available, then return to `pending`.
 
 _Leaving this stage:_
 
-- Proceeds automatically to the plan stage (**Planning** state) when it becomes the first run in the queue.
-- Can skip to completion (**Discarded** state) if a user discards it before it starts.
+- If Terraform Cloud encounters an error when fetching the configuration from VCS, the run skips to completion (**Plan Errored** state).
```

Comment:
do we want to say "completion" or .... do we want to say something that indicates it is failed. idk if completion makes it sound like the run went through, if that makes sense.

Reply:
I was shamelessly cribbing from the other states/stages on this page, which use the same "skip to completion" terminology ([plan errored](https://github.com/hashicorp/terraform-website/blob/radditude/add-config-version-archiving/content/cloud-docs/run/states.mdx?plain=1#L36), [policy check errored](https://github.com/hashicorp/terraform-website/blob/radditude/add-config-version-archiving/content/cloud-docs/run/states.mdx?plain=1#L77))! So we'll probably want to update all of them if we want to move away from "skip to completion" - maybe something like "will not continue"?

```
- If Terraform Cloud encounters an error when fetching the configuration from VCS, the run will not continue (**Plan Errored** state).
- If the user discards the run before it starts, the run will not continue (**Discarded** state).
```

Or for some other examples from further down the page:

```
- If the `terraform plan` command failed, the run will not continue (**Plan Errored** state).
- If a user canceled the plan by pressing the "Cancel Run" button, the run will not continue (**Canceled** state).

Reply:
Took a stab at rewording all the "skips to completion" on this page - how's this? [`1e316d1` (#2157)](https://github.com/hashicorp/terraform-website/pull/2157/commits/1e316d1765487c71bc28dc01c56d0eeaa72f4f77)

Reply:
I like that a lot better! It's clear now I think that the run STOPS - thanks so much for doing that!